src/Network/Network.Test/ScenarioTests/CortexTests.cs

```diff
@@ -50,5 +50,13 @@ public void TestCortexDownloadConfig()
         {
             TestRunner.RunTestScript("Test-CortexDownloadConfig");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.LiveOnly)]
```

Comment:
Why marking the test `LiveOnly`?

Reply:
Because we create storage account and create SAS and use it at run time. Previously, similar test was failing in playback and we were asked to make it liveonly, so marked this one as liveonly.